test/integration/error_output.cc

```diff
@@ -146,3 +146,76 @@ TEST(Error, ErrorOutput)
   // Check nothing has been printed
   EXPECT_TRUE(buffer.str().empty()) << buffer.str();
 }
+
+////////////////////////////////////////
+// Test Element class for sdf::Errors outputs
+TEST(ErrorOutput, ElementErrorOutput)
+{
+  std::stringstream buffer;
+  sdf::testing::RedirectConsoleStream redir(
+    sdf::Console::Instance()->GetMsgStream(), &buffer);
+
+  sdf::Errors errors;
+  sdf::ElementPtr elem = std::make_shared<sdf::Element>();
+  elem->SetName("testElement");
+
+  elem->GetAny(errors, "test");
+  EXPECT_EQ(errors[0].Code(), sdf::ErrorCode::ELEMENT_ERROR);
+  EXPECT_NE(std::string::npos, errors[0].Message().find(
+    "Unable to find value for key [test]"));
+  ASSERT_EQ(errors.size(), 1u);
```

Comment:
The `ASSERT` calls that check the size of the `errors` should come before indexing into it. Otherwise, we might get a segfault.

Reply:
True, my bad :man_facepalming:.